modules/s3/main.tf

```diff
@@ -338,6 +363,25 @@ resource "aws_cloudwatch_log_group" "supplies" {
   retention_in_days = 30
 }
 
+### Send SMS API
+resource "aws_lambda_function" "send_sms" {
```

Comment:
Please upload a dummy lambda handler so that the TF apply can run without a hitch. See here for reference https://github.com/ukraine-taskforce/infrastructure/blob/067e268fa928741bd2184dc724a1487ef8122bdf/modules/s3/main.tf#L370

Reply:
Not necessary I think I saw the corresponding .zip already in S3.
Btw by uploading a dummy lambda like this, won't the real lambda be erased every time Terraform runs afterward?...

Reply:
Nope, it won't be replaced. TF doesn't compare file hashes. If the zip exists in the bucket then the dummy is not necessary indeed.